OneginiExampleAppSwift/ViewControllers/RegisterUser/QR-code/QRCodeViewController.swift

```diff
@@ -16,9 +16,9 @@
 import AVFoundation
 import UIKit
 
-protocol QRCodeEntityProtocol {
-    var cancelled: Bool { get set }
-    var qrCodeData: String? { get set }
+protocol QRCodeViewDelegate {
+    func qrCodeView(_ qrCodeView: UIViewController, handleQRCode qrCode: String)
+    func qrCodeView(qrCodeScanCancelled qrCodeView: UIViewController)
```

Comment:
To keep the naming consistent probably it would be better to have names like `didScanQRCode` and `didCancelQRCodeScan`.